onnxscript/rewriter/rules/common/__init__.py

```diff
@@ -15,6 +15,10 @@
     "fuse_batchnorm_into_gemm_rule",
     "fuse_pad_into_conv_integer_rule",
     "fuse_pad_into_conv_rule",
+    "fuse_successive_min_rule",
+    "fuse_successive_max_rule",
+    "fuse_successive_min_max_rule",
+    "fuse_successive_max_min_rule",
```

Comment:
Following the current convention, I think we can call them
```suggestion
    "min_min_rule",
    "max_max_rule",
    "min_max_rule",
    "max_min_rule",
```